src/features/game/components/Hoarding.tsx

```diff
@@ -0,0 +1,85 @@
+import React, { useContext, useState } from "react";
+import ReCAPTCHA from "react-google-recaptcha";
+
+import close from "assets/icons/close.png";
+import token from "assets/icons/token.gif";
+
+import { Context } from "../GameProvider";
+import { useActor } from "@xstate/react";
+import { Button } from "components/ui/Button";
+import { CONFIG } from "lib/config";
+import { ITEM_DETAILS } from "../types/images";
+import { InventoryItemName } from "../types/game";
+
+export const Hoarding: React.FC = () => {
+  const { gameService } = useContext(Context);
+  const [gameState] = useActor(gameService);
+  const [showCaptcha, setShowCaptcha] = useState(false);
+  const maxedItem = gameState.context.maxedItem as InventoryItemName | "SFL";
+
+  const onCaptchaSolved = async (captcha: string | null) => {
+    await new Promise((res) => setTimeout(res, 1000));
+
+    gameService.send("SYNC", { captcha });
+  };
+
+  const makeTitle = () => {
+    const regex = new RegExp(/^[aeiou]/gi);
+    const startsWithVowel = regex.test(maxedItem);
+    const indefiniteArticle = startsWithVowel ? "an" : "a";
+    const item = maxedItem === "SFL" ? maxedItem : maxedItem.toLowerCase();
+
+    return `Are you ${indefiniteArticle} ${item} horder?!`;
+  };
+
+  const maxedItemImage =
+    maxedItem === "SFL" ? token : ITEM_DETAILS[maxedItem].image;
+
+  return (
+    <>
+      {!showCaptcha ? (
+        <div>
+          <div className="flex flex-col items-center p-1">
+            <span className="text-center text-sm sm:text-base">
+              {makeTitle()}
+            </span>
+            <img src={maxedItemImage} className="h-12 mt-2 mb-3" />
+            <p className="text-xs sm:text-sm mb-3">
+              {`I don't want to alarm you but we just heard that the Goblins have
+              noticed that you are hording a lot of this resource off chain.`}
+            </p>
+            <p className="text-xs sm:text-sm mb-1">
+              {`Word is that they're planning a raid as we speak so we recommend
+              you secure your progress on chain before continuing.`}
```

Comment:
**suggestion (non-blocking):** This is a little word, perhaps it could be shortened. Here's my suggestion, what do you think?

```suggestion
              {`The Goblins have
              noticed that you are hoarding a lot of this resource off chain.`}
            </p>
            <p className="text-xs sm:text-sm mb-1">
              {`Please sync your progress before continuing, to secure your progress on chain.`}
```

It looks like this
![hoard](https://user-images.githubusercontent.com/41215134/181443218-1bc89dd3-b6aa-452d-9f2d-15b963e16055.png)